include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -2859,6 +2859,8 @@ namespace internal
   inline std::tuple<
     const internal::MatrixFreeFunctions::ShapeInfo<VectorizedArrayType> *,
     const internal::MatrixFreeFunctions::DoFInfo *,
+    const internal::MatrixFreeFunctions::
+      MappingInfoStorage<dim - is_face, dim, Number, VectorizedArrayType> *,
```

Comment:
I must admit, I find the tuple a bit intimidating. Every addition of a value might lead to the fact that we need to update the indices in get. Furthermore, the first arguments are clear but the `unsigned int...`. I would say that a struct can be initialized the same way as a tuple, cannot it?